src/services.js

```diff
@@ -311,6 +311,24 @@ export class Services {
     return getService(win, 'story-store');
   }
 
+  /**
+   * @param {!Window} win
+   * @return {?Promise<?../extensions/amp-story/0.1/messages.MessageService>}
+   */
+  static messageServiceForOrNull(win) {
```

Comment:
Since no other extension is planning on using this service, we have no risk of race conditions. Could we just get it from the synchronous ``messageService`` method?

Reply:
Don't forget about me 🙂

Reply:
+1.  Also, this isn't stories-specific, so in theory, other people could use this for their strings too (/cc @cathyxz, does lightbox gallery have strings?)

Reply:
Nope, don't have strings on lightbox gallery (yet), but we may end up with aria strings in all components for accessibility.